facia/app/conf/HealthCheck.scala

```diff
@@ -8,3 +8,13 @@ object HealthCheck extends AllGoodHealthcheckController(9008, "/uk") {
     NoCache(if (isOk) Ok("OK") else ServiceUnavailable("Service Unavailable"))
   }
 }
+
+object HealthCheckController extends CachedHealthCheckController {
```

Comment:
all this extending works, but it would be nice if we could minimise the use of inheritance and make it more functional, just to make it easier to follow the code